From e52dfeba8ba36fc765fbe0ff0d4c93ed9aac493b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20B=C3=A9rat?= <fberat@redhat.com>
Date: Tue, 27 Jan 2026 11:31:16 +0100
Subject: [PATCH 5/6] rpm: use a '%{!r: ... }' guard around package notes
 LDFLAGS

...to prevent double inclusion when performing a relocatable link ('-r').
Relates to https://bugzilla.redhat.com/show_bug.cgi?id=2362272.
---
 rpm/redhat-package-notes.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm/redhat-package-notes.in b/rpm/redhat-package-notes.in
index 3a19b1be8a..1297ae5261 100644
--- a/rpm/redhat-package-notes.in
+++ b/rpm/redhat-package-notes.in
@@ -1,2 +1,2 @@
 *link:
-+ --package-metadata={\"type\":\"rpm\",\"name\":\"%:getenv(RPM_PACKAGE_NAME \",\"version\":\"%:getenv(RPM_PACKAGE_VERSION -%:getenv(RPM_PACKAGE_RELEASE \",\"architecture\":\"%:getenv(RPM_ARCH \",\"osCpe\":\"@OSCPE@\"}))))
++ %{!r:--package-metadata={\"type\":\"rpm\",\"name\":\"%:getenv(RPM_PACKAGE_NAME \",\"version\":\"%:getenv(RPM_PACKAGE_VERSION -%:getenv(RPM_PACKAGE_RELEASE \",\"architecture\":\"%:getenv(RPM_ARCH \",\"osCpe\":\"@OSCPE@\"}))))}
